sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -70,17 +70,17 @@ internal partial class TelemetryItem
         public TelemetryItem(string name, TelemetryItem telemetryItem, ActivitySpanId activitySpanId, ActivityKind kind, DateTimeOffset activityEventTimeStamp) :
                         this(name, FormatUtcTimestamp(activityEventTimeStamp.DateTime))
         {
-            Tags[ContextTagKeys.AiOperationParentId.ToString()] = activitySpanId.ToHexString();
-            Tags[ContextTagKeys.AiOperationId.ToString()] = telemetryItem.Tags[ContextTagKeys.AiOperationId.ToString()];
+            Tags[ContextTagKeys.AiOperationParentId.ToString()] = activitySpanId.ToHexString().Truncate(SchemaConstants.Tags_AiOperationParentId_MaxLength);
+            Tags[ContextTagKeys.AiOperationId.ToString()] = telemetryItem.Tags[ContextTagKeys.AiOperationId.ToString().Truncate(SchemaConstants.Tags_AiOperationId_MaxLength)];
```

Comment:
```suggestion
            Tags[ContextTagKeys.AiOperationId.ToString()] = telemetryItem.Tags[ContextTagKeys.AiOperationId.ToString()].Truncate(SchemaConstants.Tags_AiOperationId_MaxLength);
```
probably meant to do this ^?

Reply:
good catch. thank you